auth-web/package.json

```diff
@@ -56,7 +56,7 @@
     "stylus": "^0.54.7",
     "stylus-loader": "^3.0.2",
     "ts-jest": "^23.0.0",
-    "typescript": "^3.2.1",
+    "typescript": "^3.7.2",
```

Comment:
Please confirm that this will not generate any additional type errors in the solution - that's a big version jump.